packages/sirius-web/frontend/sirius-web-application/src/diagrams/useDiagramFilterSubscription.tsx

```diff
@@ -59,20 +60,19 @@ export const useDiagramFilterSubscription = (
 
   const variables: GQLDiagramFilterEventVariables = { input };
 
+  const onComplete = () => setState((prevState) => ({ ...prevState, complete: true }));
+
   const onData = ({ data }: OnDataOptions<GQLDiagramFilterEventSubscription>) => {
     const { data: gqlDiagramFilterEventSubscription } = data;
     if (gqlDiagramFilterEventSubscription) {
       const { diagramFilterEvent: payload } = gqlDiagramFilterEventSubscription;
       if (isFormRefreshedEventPayload(payload)) {
-        const { form } = payload;
-        setState((prevState) => ({ ...prevState, form, complete: false }));
+        setState((prevState) => ({ ...prevState, complete: false }));
```

Comment:
You can remove the rest of the code from this onData, we don't anything other than "if we receive some data, then it's not complete anymore", we don't need to do that only on a form refreshed event payload